x-pack/test/api_integration/apis/metrics_ui/snapshot.ts

```diff
@@ -159,7 +295,7 @@ export default function ({ getService }: FtrProviderContext) {
       after(() => esArchiver.unload('x-pack/test/functional/es_archives/infra/7.0.0/hosts'));
```

Comment:
Do we really need to run a test against 7.0.0?

Reply:
Reminds me of [this change](https://github.com/elastic/kibana/issues/131152#issuecomment-1149223980) where we wanted to use ECS fields that didn't exist in previous versions and didn't know if there was an actual cutoff point for older data compatibility.  I don't think it hurts to leave this here because then we would at least know that v7 data wouldn't work with our API and perhaps we could do something differently to avoid that.